### PR TITLE
Add governance doc on Tech Leads

### PR DESCRIPTION
**Pull request by lianmakesthings**
Clearly outline rights & responsibilities of a Tech Lead & the nomination process

### CODE REVIEW

File: governance/tech-leads.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+### Technical Lead Trial Run
+To ensure that the nominee is a good fit for the role, TAG leadership may decide to offer a trial run of 3 months to a promising candidate. During this time, the nominee will be given low-stakes responsibilities to demonstrate their capabilities and commitment to the role. At the end of the trial run, the internal TAG vote will be held.
+
+## Technical Lead nomination requirements
+
+Eligible nominees should have experience with some TAG groups (such as WGs) and have actively contributed in a meaningful way. Potential nominees must accept the nomination.
+
+If a given nominee does not meet all requirements, the Co-Chairs may still review the nomination to determine if enough intent and commitment is present to move forward with an endorsement. This discussion will usually take place during the TAG's internal voting process.
+
+A non-exhaustive list of activities an eligible candidate may have participated in:
```
Comment (schristoff) [inline governance/tech-leads.md:29]: I really like this, as my first question was "what is a meaningful way" mean.

**Comment by schristoff** [comment]
Is #454 maybe an older version of this? It would be great to include some tidbits from there perhaps - like discussing about Emeritus status 🤔

**Comment by lianmakesthings** [comment]
> Is #454 maybe an older version of this? It would be great to include some tidbits from there perhaps - like discussing about Emeritus status 🤔

Yes I used that as a blueprint but focused on the Tech Leads for this PR. 
Good point about the comments & tidbits, I didn't even think to check them. I'll include some of them here if they make sense, but will need to be careful to paste someone else's comments/requests as it may be unclear how/when they are resolved.

**Review by roberthstrand — APPROVED** [review]
This seems to outline the process in a comprehensive and well written way. LGTM 👏

**Comment by thschue** [comment]
Looks good! I like the "trial-run" idea.

**Review by thschue — APPROVED** [review]

**Pull request merged**